index.html

```diff
@@ -101,35 +101,115 @@ <h2>
-          <code>browser</code>.
+          Additionally to the normal
+          <a data-cite="appmanifest#dfn-display-modes-list">display modes</a>,
+          [=manifest/display_override=] also supports certain extensions to it.
         </p>
-        <pre class="example json" title="Advanced display usage manifest">
-          {
-            "name": "Recipe Zone",
-            "description": "All of the recipes!",
-            "icons": [{
-              "src": "icon/hd_hi",
-              "sizes": "128x128"
-            }],
-            "start_url": "/index.html",
-            "display_override": ["minimal-ui"],
-            "display": "standalone",
-            "theme_color": "yellow",
-            "background_color": "red"
-          }
-        </pre>
+        
+        <dt>
+          <dfn data-export="" data-dfn-for="display mode">
+            borderless
+          </dfn>
+        </dt>
+        <dd>
+          The web application does not have any host-native title bar or
+          [=window controls=] visible and with the web contents extended to the
+          whole title bar area. The app can specify [=draggable region=]s in the
+          web contents to create a customized title bar. The user agent may
+          change the title bar state depending on various security
+          considerations, like an out-of-scope navigation.
+        </dd>
+
+        <dt>
+          [=display mode/window-controls-overlay=]
+        </dt>
+
+        <section class="informative">
+          <h4>
+            [=manifest/display_override=] usage example
+          </h4>
+          <p>
+            The following shows a [=manifest=] that prefers the
+            <code>minimal-ui</code> <a data-cite=
+            "appmanifest#dfn-display-mode">display mode</a> over
+            <code>standalone</code>, but if <code>minimal-ui</code> isn't
+            supported, falls back to <code>standalone</code> as opposed to
+            <code>browser</code>.
+          </p>
+          <pre class="example json" title="display_override usage manifest">
+            {
+              "name": "Recipe Zone",
+              "description": "All of the recipes!",
+              "icons": [{
+                "src": "icon/hd_hi",
+                "sizes": "128x128"
+              }],
+              "start_url": "/index.html",
+              "display_override": ["minimal-ui"],
+              "display": "standalone",
+              "theme_color": "yellow",
+              "background_color": "red"
+            }
+          </pre>
+        </section>
+      </section>
+      <section>
+        <h3>
+          Defining <dfn data-export="" data-lt="draggable region">draggable
+            regions</dfn>
+        </h3>
+        <p>
+          To define which regions or elements in for example a title bar are
+          draggable, this specification defines the CSS
+          <dfn>`app-region`</dfn> property.
+        </p>
+        <p>
```

Comment:
This is a weird paragraph to have _in the standard_. We are defining `app-region` right here, so we shouldn't say right below that we're recommending `-webkit-app-region`.

I would say this should be turned into a non-normative note, but I think even that goes too far: we should not recommend webkit-prefixed things in a standard.

Perhaps as a compromise, we could turn this para into a `class="issue"` and say "At-risk feature: `app-region` has not been implemented in any user agent, so is at risk. Note that some user agents use the non-standard `-webkit-app-region` for the same purpose." (Note: This doesn't say WAR "shall be used as the supported"; it just points out the fact that it exists.)

Reply:
PTAL if this'd be good:

![image](https://github.com/WICG/manifest-incubations/assets/46299415/06b81666-a552-4ea8-86c5-300cb8ddc7a4)

Reply:
optional nit to do the definition separate from the issue block. (so the second P and points occur separatly, and the app-region is 'defined' there, and then the first P is kept in the issue block). But I'm also fine as-is, having the whole thing as an issue communicates this probably better.

Reply:
Done